### PR TITLE
Roberto/and 1426

### PR DESCRIPTION
**Pull request by robertogdsbc**
feat(HomeBrew): AND-1426

### CODE REVIEW

File: morph/morphdevapp/src/main/java/com/blockchain/morph/dev/MainActivity.kt [hunk 1]
```diff
@@ -13,6 +14,7 @@ class MainActivity : AppCompatActivity() {
     }
 
     fun launchExchange(view: View) {
-        startActivity(ExchangeActivity.intent(this, "GBP"))
+        val intent = Intent(this, ExchangeHistoryActivity::class.java)
```
Comment (ditn) [inline morph/morphdevapp/src/main/java/com/blockchain/morph/dev/MainActivity.kt]: IMO in the future we should delegate this to `MorphActivityLauncher`. Here we can check for completed transactions and direct people straight to a new exchange if there are no transactions, or to transaction history. For now this is fine though.

File: morph/ui/src/main/java/com/blockchain/koin/morphUiModule.kt [hunk 2]
```diff
@@ -16,18 +17,19 @@ val morphUiModule = applicationContext {
             override fun startForResult(parent: Activity): Int {
                 val requestCode = requestCode++
                 UsStateSelectionActivity.startForResult(
-                    parent,
-                    requestCode
+                        parent,
```
Comment (ditn) [inline morph/ui/src/main/java/com/blockchain/koin/morphUiModule.kt]: This whitespace concerns me. Let's double check your style tomorrow.

File: morph/ui/src/main/res/drawable/trade_status_failed_circle.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <item>
+        <!--
+            android:shape
+                Keyword. Defines the type of shape. Valid values are:
+
+                    rectangle : A rectangle that fills the containing View. This is the default shape.
+                    oval : An oval shape that fits the dimensions of the containing View.
+                    line : A horizontal line that spans the width of the containing View. This
+                           shape requires the <stroke> element to define the width of the line.
+                    ring : A ring shape.
```
Comment (ditn) [inline morph/ui/src/main/res/drawable/trade_status_failed_circle.xml]: Unnecessary comment

File: morph/ui/src/main/java/com/blockchain/koin/morphUiModule.kt (hunk 2 above)
Comment (robertogdsbc) [reply morph/ui/src/main/java/com/blockchain/koin/morphUiModule.kt]: Just realised I did a full re-install of android studio and didn't configure the proper style

File: morph/morphdevapp/src/main/java/com/blockchain/morph/dev/MainActivity.kt (hunk 1 above)
Comment (robertogdsbc) [reply morph/morphdevapp/src/main/java/com/blockchain/morph/dev/MainActivity.kt]: Agreed

File: quality/jacocoLibrary.gradle [hunk 4]
```diff
@@ -1,3 +1,5 @@
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-android'
```
Comment (ditn) [inline quality/jacocoLibrary.gradle]: These need removing again!

**Pull request merged**